src/js/components/FileInput/FileInput.js

```diff
@@ -214,49 +261,77 @@ const FileInput = forwardRef(
                     inputRef.current.focus();
                   }}
                 />
+                {files.length < 2 && (
```

Comment:
When is this use case of <2 ? I guess I would've assumed it was either `<1` or `<=2` if this is referring to multiple files

Reply:
yeah I think in the beginning I was going to use this for both `file===1 and !files` However now Im only needing this button when  `file ===  1` so I just  changed that!